docs/design/solutions/pce-weights-strategies/pce-weights-strategies.md

```diff
@@ -0,0 +1,19 @@
+# Refactor PCE with weights computation strategies
+
+## Goals
```

Comment:
I suggest starting with an example, and possibly a better one than below. The current set of Goals are okay at a very high level, but there isn't sufficient information on either the complete set of problems that need to be solved, or the set of solutions that would work.

Reply:
I see. Added some explanation about.